external/stdk/Kconfig

```diff
@@ -15,6 +15,12 @@ config STDK_IOT_CORE
 
 if STDK_IOT_CORE
 
+config STDK_CURRENT_TIME_YEAR
```

Comment:
This works, but a bit confusing.
As discussed just now, please discuss with @SeonghoByeon on new solution.

Reply:
yes, this Kconfig also seems weired, I will remove this and try to check the time with initialized value.